src/Onboarding/DefaultFormFactory.php

```diff
@@ -0,0 +1,185 @@
+					'_give_anonymous_donation'             => 'global',
+					'_give_donor_comment'                  => 'global',
+					'_give_logged_in_only'                 => 'enabled',
+					'_give_show_register_form'             => 'none',
+					'_give_goal_option'                    => 'disabled',
+					'_give_goal_format'                    => 'amount',
+					'_give_set_goal'                       => 1,
+					'_give_number_of_donor_goal'           => 1,
+					'_give_goal_color'                     => $this->primaryColor,
+					'_give_close_form_when_goal_achieved'  => 'disabled',
+					'_give_form_goal_achieved_message'     => __( 'Thank you to all our donors, we have met our fundraising goal.', 'give' ),
+					'_give_terms_option'                   => 'global',
+					'_give_agree_label'                    => '',
+					'_give_agree_text'                     => '',
+					'give_stripe_per_form_accounts'        => 'disables', // Note: Doesn't use underscore prefix.
+					'_give_default_stripe_account'         => '', // @TODO Might need to update this???
+					'_give_email_options'                  => 'global',
+					'_give_email_template'                 => 'default',
+					'_give_email_logo'                     => '',
+					'_give_from_name'                      => get_bloginfo( 'name' ),
+					'_give_from_email'                     => get_bloginfo( 'admin_email' ),
+					'_give_new-donation_notification'      => 'global',
+					'_give_new-donation_email_subject'     => sprintf( '%s - #{payment_id}', __( 'New Donation', 'give' ) ),
+					'_give_new-donation_email_header'      => __( 'New Donation!', 'give' ),
+					'_give_new-donation_email_message'     => give_get_default_donation_notification_email(),
+					'_give_new-donation_email_content_type' => 'text/html',
+					'_give_new-donation_recipient'         => [
+						'email' => get_bloginfo( 'admin_email' ),
+					],
+					'_give_donation-receipt_notification'  => 'global',
+					'_give_donation-receipt_email_subject' => __( 'Donation Receipt', 'give' ),
+					'_give_donation-receipt_email_header'  => __( 'Donation Receipt', 'give' ),
+					'_give_donation-receipt_email_mesage'  => give_get_default_donation_receipt_email(),
+					'_give_donation-receipt_email_content_type' => 'text/html',
+					'_give_form_goal_progress'             => -1,
+				],
+			]
+		);
+
+		return $formID;
+	}
+
+	/**
+	 * Default values extracted from src/Views/Form/Templates/Sequoia/optionConfig.php
+	 *
+	 * Updates the default primary_color to match the Onboarding Wizard.
+	 *
+	 * @return array
+	 *
+	 * @since 2.8.0
+	 */
+	public function getTemplateConfig() {
+		return [
+			'introduction'        => [
+				'enabled'       => 'enabled',
+				'headline'      => __( 'Support Our Cause', 'give' ),
+				'description'   => __( 'Help our organization by donating today! All donations go directly to making a difference for our cause.', 'give' ),
+				'image'         => '', // @TODO Add a default image.
```

Comment:
Need to add a default image to the default form.

Reply:
Resolved by opening #5025.

Reply:
Nevermind. I was able to add the default image via a static file, instead of inserting into the media library.